static/app/components/events/interfaces/sourceMapsDebuggerModal.tsx

```diff
@@ -482,10 +482,25 @@ export function SourceMapsDebuggerModal({
       <Body>
         <p>
           {t(
-            "It looks like the original source code for this stack frame couldn't be determined when this error was captured. To get the original code for this stack frame, Sentry needs source maps to be configured."
+            'It looks like the original source code for this stack frame. To get the original code for this stack frame, Sentry needs source maps to be uploaded.'
           )}
         </p>
-        {metaFrameworksWithSentryWizardInOnboarding.includes(platform) ? (
+        {isReactNativeSDK({
+          sdkName: sourceResolutionResults.sdkName,
+        }) ? (
+          <Fragment>
+            <WizardInstructionParagraph>
+              {tct(
+                "For React Native projects, source maps should be generated and uploaded automatically during the build process. If your source maps aren't showing up, consult our [link:React Native source maps documentation].",
+                {
+                  link: (
+                    <ExternalLinkWithIcon href="https://docs.sentry.io/platforms/react-native/sourcemaps/" />
+                  ),
+                }
+              )}
+            </WizardInstructionParagraph>
+          </Fragment>
```

Comment:
`Fragment` is only used to wrap multiple children. We have only 1 in this case, so we can remove it 
```suggestion
         <WizardInstructionParagraph>
              {tct(
                "For React Native projects, source maps should be generated and uploaded automatically during the build process. If your source maps aren't showing up, consult our [link:React Native source maps documentation].",
                {
                  link: (
                    <ExternalLinkWithIcon href="https://docs.sentry.io/platforms/react-native/sourcemaps/" />
                  ),
                }
              )}
            </WizardInstructionParagraph>
```